cmd/keytransparency-client/cmd/cache.go

```diff
@@ -1,44 +0,0 @@
-// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Trillian log does not support SignedCertificateTimestamps (SCTs). Previously, SCTs needed to be saved to disk and verified later. Now all verification is immediate, thus no need to verify previously retrieved results anymore.